docs/UserGuide.md

```diff
@@ -102,16 +102,16 @@ Edits an existing elderly in ContactMate.
 
 Format: `edit INDEX/NRIC [i/NRIC] [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [c/CALL_FREQUENCY] [t/TAG]…​`
 
-* Edits the elderly at the specified `INDEX` or `NRIC`. The index refers to the index number shown in the displayed person list.The index must be within the range of list shown and **must be a positive integer** 1, 2, 3, …​
+* Edits the elderly at the specified `INDEX` or `NRIC`. The index refers to the index number shown in the displayed person list. The index must be within the range of list shown and **must be a positive integer** 1, 2, 3, …​
 * The `NRIC` has to be a valid NRIC.
 * At least one of the optional fields must be provided.
-* Existing values will be updated to the input values.
+* The existing values will be replaced with the new input values.
 * When editing tags, the existing tags of the elderly will be removed i.e adding of tags is not cumulative.
 * You can remove all the elderly’s tags by typing `t/` without
     specifying any tags after it.
 
 Examples:
-*  `edit 1 i/S8340008J p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st elderly to be `91234567` and `johndoe@example.com` respectively.
+*  `edit 1 i/S8340008J p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st elderly shown in the list to be `91234567` and `johndoe@example.com` respectively.
```

Comment:
I think this should edit the nric of the elderly too?

Reply:
Yes, let me add the NRIC too